integration/test_aggregation_with_redis.py

```diff
@@ -0,0 +1,1854 @@
+import asyncio
```

Comment:
Would be good to avoid duplicating this giant test file.

Reply:
Right. Would do as a lower priority

Reply:
Done.